Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -127,7 +127,6 @@ div.bocList table.bocListTable td.bocListTitleCell > span
 }
 
 div.bocList table.bocListTable td.bocListDataCell > a:not([hidden]),
-div.bocList table.bocListTable td.bocListDataCell > span:not([hidden]),
 div.bocList table.bocListTable td.bocListDataCell > div:not([hidden])
```

Comment:
:D And you wonder why that was never a red flag :D

Reply:
I dont think I understand ;/